docs/diagrams/StorageClassDiagram.puml

```diff
@@ -38,6 +39,7 @@ JsonUserPrefsStorage .up.|> UserPrefsStorage
 JsonAddressBookStorage .up.|> AddressBookStorage
 JsonAddressBookStorage ..> JsonSerializableAddressBook
 JsonSerializableAddressBook --> "*" JsonAdaptedPerson
-JsonAdaptedPerson --> "*" JsonAdaptedTag
+JsonAdaptedPerson --> "*" JsonAdaptedPastAppointment
+JsonAdaptedPerson --> "*" JsonAdaptedMedication
```

Comment:
Maybe making this arrow longer would help condense the diagram width a bit, but otherwise looks good!